packages/activity/src/index.ts

```diff
@@ -26,7 +26,13 @@
  * @module
  */
 
-import { msToNumber } from '@temporalio/internal-workflow-common';
+if ((globalThis as any).__TEMPORAL__ !== undefined) {
+  throw new IllegalStateError(
+    "You are importing from '@temporalio/activity' in your Workflow code, which doesn't work. Workflow code should only import from '@temporalio/workflow' and '@temporalio/common'."
```

Comment:
How about something like:
```suggestion
    "Importing from '@temporalio/activity' from Workflow code is not supported. Workflow code should only import from '@temporalio/workflow' and '@temporalio/common'. More information here: https://docs.temporal.io/docs/typescript/determinism/"
```

Reply:
I'm not a fan of this personal tone in error messages